src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -155,6 +156,49 @@ public function thereIsAnAdministratorIdentifiedBy($email)
     }
 
     /**
+     * @Given hers account is not verified
+     * @Given I have not verified my account (yet)
+     */
+    public function accountIsNotVerified()
+    {
+        $user = $this->sharedStorage->get('user');
+        $user->setVerifiedAt(null);
+
+        $this->userManager->flush();
+    }
+
+    /**
+     * @Given (I have)(he has) already received a verification email
```

Comment:
Shouldn't be ``(?:I have|he has)``? Also ``I`` or ``he`` can be used to return ``'user'`` from shared storage, wdyt?